mmdet3d/models/segmentors/base.py

```diff
@@ -63,18 +69,18 @@ def forward(self, return_loss=True, **kwargs):
     def show_results(self,
                      data,
                      result,
-                     palette=None,
```

Comment:
Will change the order of argument cause BC-breaking?

Reply:
I think it won't. Because this function is only called inside the codebase of MMDet3D (at least we don't have instructions that require calling this function).